modules/dicom_archive/test/TestPlan

```diff
@@ -20,15 +20,17 @@
 6-  Under 'View Details' Check to make sure all the specific parameters are Populated
     [Manual Testing]
 7-  Under 'View Details' Click on 'Acquisition ID' to view the violated scans (if there are any) if permission 
-'Violated Scans: View all-sites Violated Scans' is granted
+    'Violated Scans: View all-sites Violated Scans' is granted
     [Manual Testing]
-8-  Under 'View Details' Click on 'Show/Hide Files' to view a list of all Dicom Files for the given patient
+8-  Under 'View Details' Click on 'Show/Hide Series' and 'Show/Hide Files' to view a list of all Dicom Series/Files,
+    respectively, for the given patient
     [Manual Testing]
-9- Under 'View Details' Click on 'Show/Hide metadata' to view the details of the metadata file
+9-  Make sure that the value of 'Patient ID' column is 'INVALID-HIDDEN' if the patient-name does not match the 
```

Comment:
I would rephrase this point like that:

Make sure that the value of **the** 'Patient ID' column is set to 'INVALID-HIDDEN' if the **Patient ID** does not match the regex in the Configuration module under the 'Imaging Modules' submenu. **Repeat the same operation for 'Patient Name'.**